### PR TITLE
[cleanup] mark DUNE build as experimental.

### PR DESCRIPTION
**Pull request by dr-robertk**
This PR fixes issue #261 .

### CODE REVIEW

File: INSTALL [hunk 1]
```diff
@@ -45,32 +46,8 @@ installing DUNE manually [1].
...
-  cd $EWOMS_SOURCE_DIR/build-cmake
-  make lens_immiscible_ecfv_ad
-
-to compile the `lens_immiscible_ecfv_ad` executable. Compiling all
-available tests can be achieved via
-
-  cd $EWOMS_SOURCE_DIR/build-cmake
-  make build_tests
+Build instructions for eWoms using the OPM build system are to be found on the
+OPM web page [1].
```
Comment (andlaus) [inline INSTALL]: I would like to have the alternative build option to be mentioned here. E.g. `If these guidlines do not work for you, an alternative way to build eWoms is that is based on the default DUNE build procedure is available, see the "Alternative build system" section below.`

File: INSTALL [hunk 2]
```diff
@@ -114,6 +91,40 @@ argument, e.g.,
   cd $EWOMS_SOURCE_DIR/build-cmake
   ./bin/lens_immiscible_ecfv_ad --help
 
+Alternative build system
+========================
+
+An experimental version making use of the DUNE build system can be found in the
```
Comment (andlaus) [inline INSTALL]: please replace with "An experimental way of building eWoms making use of..."

File: INSTALL [hunk 3]
```diff
@@ -114,6 +91,40 @@ argument, e.g.,
...
+opm-dune-cmake module from [3] and is to be placed next to the OPM and DUNE
+modules.
+
+The easiest way to compile eWoms is to follow the generic installation
```
Comment (andlaus) [inline INSTALL]: The easiest way to compile eWoms *this way* is to ...

**Comment by dr-robertk** [comment]
Should be good to be merged now.

**Comment by andlaus** [comment]
I agree. thanks!

**Pull request merged**